### PR TITLE
Add layout to the page

### PR DESCRIPTION
**Pull request by BeJuanitox**
image
image
image
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,16 +6,35 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="styles.css">
+  <link rel="icon" href="./images/Resume.ico">
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Prompt:wght@300&display=swap');
+  </style>
```
Comment (Andres2D) [inline index.html]: Put this in the styles.css file.

File: index.html [hunk 2]
```diff
@@ -6,16 +6,35 @@
...
 
 <body>
-  <section class="header">
-    <img class="picture" src="./images/Sebastian.jpg" alt="profile">
-    <h1 class="name">Sebastian Morales</h1>
-    <div class="about">
-      <h2 class="description">Welcome, I'm Sebastian Morales, I'm currently a student of Software Engineering, I seek to
-        improve more every day, I enjoy playing videogames, listening music and learning new things.</h2>
+  <header>
+    <div class="headerContainer">
```
Comment (Andres2D) [inline index.html]: Do you really need thid div tag?, the class could be placed in the header tag and delete the div.

File: index.html [hunk 3]
```diff
@@ -6,16 +6,35 @@
...
+      <img class="headerPicture" src="./images/Sebastian.jpg" alt="profile">
+      <h2 class="headerDescription">Fight for your dreams and try to be better every day.</h2>
+      <button onclick="redirectButton()" id="button">Look</button>
```
Comment (Andres2D) [inline index.html]: This id `button` is a bad name, try using another name more specific.

File: index.html [hunk 4]
```diff
@@ -204,23 +223,25 @@ <h2 class="name">Web Page</h2>
...
+        <i class="fa-brands fa-telegram itemSocial itemTlg"></i>
+      </a>
+    </div>
+  </footer>
   <script src="https://kit.fontawesome.com/7851ea6f75.js" crossorigin="anonymous"></script>
   <script defer src="scripts.js"></script>
 </body>
 
-</html>
+</html>
```
Comment (Andres2D) [inline index.html]: Add empty space at the bottom.

File: styles.css [hunk 5]
```diff
@@ -1,38 +1,116 @@
...
+
+header {
+  width: 100%;
+  height: 100vh;
+  background-color: var(--bg-header);
 }
 
-.header {
+html {
+  scroll-behavior: smooth;
```
Comment (Andres2D) [inline styles.css]: I think that this could be adde in the body tag.

File: styles.css [hunk 6]
```diff
@@ -245,6 +341,11 @@ svg {
   font-size: 25px;
 }
 
+footer {
+  margin-top: auto;
```
Comment (Andres2D) [inline styles.css]: This is auto by default.

File: styles.css [hunk 7]
```diff
@@ -253,9 +354,24 @@ svg {
...
+  margin-top: 50px;
+  margin-bottom: 25px;
+  color: white;
+  font-weight: 500;
+}
+
 .social {
   border: 2px solid var(--clr-border2);
   border-radius: 50px;
+  background: #CDCDCD;
```
Comment (Andres2D) [inline styles.css]: Don´t use var color for some properties and hex values for others. Try to keeo the standar.

File: scripts.js [hunk 8]
```diff
@@ -7,61 +7,65 @@ let java = document.getElementById('java');
...
     counterjava++;
     java.innerHTML = `${counterjava}%`;
   }
-  
+
 }, 80)
+
+function redirectButton() {
+  location.href = "#start";
+}
```
Comment (Andres2D) [inline scripts.js]: Is this actually working?

**Review by Andres2D — CHANGES_REQUESTED** [review]

File: styles.css (hunk 5 above)
Comment (BeJuanitox) [reply styles.css]: Isn't possible for correct operation

File: scripts.js (hunk 8 above)
Comment (BeJuanitox) [reply scripts.js]: Yes

Comment (Andres2D) [inline scripts.js]:
```suggestion
const redirectButton = () => {
  location.href = "#start";
}
```
Use a the modern arrow function.

**Review by Andres2D — APPROVED** [review]
Well done.

**Pull request merged**